README.md

```diff
@@ -16,6 +22,62 @@ github-issues fetch ustwo/ustwo.com-frontend \
 ```
 
 
+### Upload issues
+
+Create a file `myissues.csv` with the following format:
```

Comment:
It might be worth having a command that initialises a new CSV file with the correct format. It would reduce the barrier to entry for using the new upload functionality.